cmd/juju/cloud/show_test.go

```diff
@@ -55,24 +85,26 @@ clouds:
         endpoint: http://london/1.0
     config:
       bootstrap-timeout: 1800
```

Comment:
The last change is made, add bootstrap-config and model-config to region-config output.  Updated TestShowWithRegionConfig() to test all of it together.